src/config/entities/__tests__/configuration.ts

```diff
@@ -39,4 +39,7 @@ export default (): ReturnType<typeof configuration> => ({
   safeTransaction: {
     useVpcUrl: false,
   },
+  features: {
+    humanDescription: true,
```

Comment:
I'd let this as false by default and enable it on-demand in each related test, wdyt?